lib/commands/waitForExist.js

```diff
@@ -65,7 +65,9 @@ module.exports = function waitForExist(selector, ms, reverse) {
 
                 var now = new Date().getTime();
                 if(now - start > ms) {
-                    return cb(null, false);
```

Comment:
dang, yeah totally right. `waitForExist` definitely should throw if no element was found (or an element was found in reverse mode. With my [recent commit](https://github.com/webdriverio/webdriverio/commit/7bca59b165b24e8fde3b7c40988e679380ede489) this should work. Thanks for clarifying.